starter-code/css/main.css

```diff
@@ -8,3 +8,91 @@ body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
 }
+
+
+.logo{
+
+  background-color: #4881ab; 
+  height: 200px;
+  /* text-align: right; */
+  
+  font-family: Billabong;
+ 
+}
+
+
+
+.logo span {
+  position: relative;
+  top:20px;
+  right: 240px;  
+  margin-left: 900px;
+  font-size: 20px;
+}
+
+.logo2  {
+  background-color: #4881ab;
+  position: absolute;
+  top:45px;
+  right: 300px;  
+  margin-left: 900px;
+  font-size: 20px;
+}
+
+.logo2 log{
+  background-color: #4881ab;
+
+}
+
+
+
+  
+
+
+.footer{
+
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 1rem;
```

Comment:
again review what position: absolute does. the footer should be moved up some more.